diff --git a/node_modules/react-quill/lib/index.js b/node_modules/react-quill/lib/index.js
index 9b75a6a..e04a43b 100644
--- a/node_modules/react-quill/lib/index.js
+++ b/node_modules/react-quill/lib/index.js
@@ -282,10 +282,7 @@
 	},
 
 	getEditor: function(){
-		if (!this.quill) return;
-		// using findDOMNode because of https://github.com/facebook/react/issues/3587
-		return this.quill.root;
+		return this.quill;
 	},
 
 	getEditingArea: function(){
@@ -433,7 +430,7 @@
 	},
 
 	focus: function(){
-		this.quill.focus();
+		this.quill.focus()
 	},
 
 	blur: function(){
@@ -482,4 +479,4 @@
 	displayName: 'React Quill'
 };
 
-module.exports = ReactQuill;
\ No newline at end of file
+module.exports = ReactQuill;